src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralServiceTest.kt

```diff
@@ -220,6 +220,23 @@ class ReferralServiceTest @Autowired constructor(
     referralService.updateDraftReferral(referral, DraftReferralDTO(hasAdditionalResponsibilities = true, whenUnavailable = "wednesdays"))
   }
 
+  @Test
+  fun `when usingRarDays is true, maximumRarDays must be set`() {
+    val referral = Referral()
+    // this is fine
```

Comment:
Is there a negative variant for assertThrows? It may replace the comment if we could say `assertDoesNotThrow<ValidationError>` or similar 🤔 

(I'm a fan of self-explaining assertions as I always forget to update the comments 😅)